### ISSUE TITLE
WORDPRESS CHART custom HTACCESS does not work with 5.8.2 wordpress image

### ISSUE THREAD

**Issue by zzOzz** [bitnami/charts#8120]
**Which chart**:

WORDPRESS CHART

**Describe the bug**

Custom HTACCESS not loaded with new wordpress image

works with bitnami/wordpress:5.8.1-debian-10-r10  
doesn't work with  bitnami/wordpress:5.8.2-debian-10-r1

**To Reproduce**

create deployment
with the following values 

~~~
allowOverrideNone: false
customHTAccessCM: htaccess-config
htaccessPersistenceEnabled: false
~~~
and a confimap named  htaccess-config

ex:
~~~
apiVersion: v1
data:
  wordpress-htaccess.conf: |-
    LoadModule remoteip_module modules/mod_remoteip.so
    RemoteIPHeader X-Forwarded-For
    RemoteIPInternalProxy 10.0.0.0/8
    RewriteEngine on
    RewriteCond %{HTTP:Authorization} ^(.*)
    RewriteRule ^(.*) - [E=HTTP_AUTHORIZATION:%1]
    SetEnvIf Authorization "(.*)" HTTP_AUTHORIZATION=$1
~~~
**Expected behavior**

the generated /opt/bitnami/apache/conf/vhosts/wordpress-vhost.conf doesn't contains this line:

 Include "/opt/bitnami/apache/conf/vhosts/htaccess/wordpress-htaccess.conf"
~~~
<VirtualHost 127.0.0.1:8080 _default_:8080>
  ServerAlias *
  DocumentRoot /opt/bitnami/wordpress
  <Directory "/opt/bitnami/wordpress">
    Options -Indexes +FollowSymLinks -MultiViews
    AllowOverride All
    Require all granted
    # BEGIN WordPress fix for plugins and themes
    # Certain WordPress plugins and themes do not properly link to PHP files because of symbolic links
    # https://github.com/bitnami/bitnami-docker-wordpress-nginx/issues/43
    RewriteEngine On
    RewriteRule ^bitnami/wordpress(/.*) $1 [L]
    # END WordPress fix for plugins and themes
    # BEGIN WordPress
    # https://wordpress.org/support/article/htaccess/#basic-wp
    RewriteEngine On
    RewriteRule .* - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization}]
    RewriteBase /
    RewriteRule ^index\.php$ - [L]
    RewriteCond %{REQUEST_FILENAME} !-f
    RewriteCond %{REQUEST_FILENAME} !-d
    RewriteRule . /index.php [L]
    # END WordPress
  </Directory>
### MISSING LINE Include "/opt/bitnami/apache/conf/vhosts/htaccess/wordpress-htaccess.conf"
</VirtualHost>
~~~

**Comment by marcosbc** [comment]
Thanks for reporting this @zzOzz! You are totally right, there is currently a bug in the logic of the container solution, and we are already looking into fixing this ASAP. We expect a fixed version to be available soon (probably tomorrow).

**Comment by marcosbc** [comment]
Hi @zzOzz, we just released version 12.1.29 of this chart, which fixes these issues. The version of the container image with the fix is 5.8.2-debian-10-r5.

**Comment by zzOzz** [comment]
Thanks a lot, I've checked the new image with the old wordpress-12.1.28 chart and it's working again !!

~~~
helm upgrade msh-sites-dev bitnami/wordpress --reuse-values --set-string image.tag="5.8.2-debian-10-r5"
~~~
